pkg/cmd/reset.go

```diff
@@ -90,5 +99,23 @@ func runReset(opts *resetOpts) error {
 		return errors.Wrap(err, "failed to initialize State")
 	}
 
+	fmt.Println("The following nodes will be resetted: ")
+	for _, node := range s.Cluster.ControlPlane.Hosts {
+		fmt.Printf("\t+ reset control plane node %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+	for _, node := range s.Cluster.StaticWorkers.Hosts {
+		fmt.Printf("\t+ reset static worker nodes %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
```

Comment:
Would be also good to list the Nodes that are managed by the machine-controller and will be destroyed before the cluster

Reply:
@kron4eg  do you have a hint on how I would do this? Do I need the `LiveCluster` API for this?

Reply:
@shibumi you will need to:
* get the list of nodes
* every MC-managed Node has Machine object reference in its metadata. If it doesn't it means — node is not managed by the machine-controller

https://github.com/kubermatic/kubeone/blob/168338870a5eb01a4c4d2831ad84ac415ccd479e/pkg/templates/machinecontroller/helper.go#L104-L110

Reply:
@kron4eg do you want to have another look on my current solution? I have the feeling this should work., but I want to be sure it's okay.